packages/components/notification/stories/Notification.stories.tsx

```diff
@@ -0,0 +1,178 @@
+import React, { useState } from 'react';
+
+import { NotificationItem, Notification } from '../src';
+import type { Placement } from '../src/NotificationsManager';
+import { Button, ButtonGroup } from '@contentful/f36-button';
+import { action } from '@storybook/addon-actions';
+import { Flex } from '@contentful/f36-core';
+import { SectionHeading } from '@contentful/f36-typography';
+
+let index = 0;
+const getUniqueNumber = () => {
+  index += 1;
+  return index;
+};
+
+export default {
+  component: NotificationItem,
+  parameters: {
+    propTypes: NotificationItem['__docgenInfo'],
+  },
```

Comment:
I think we don't need this anymore

These was a hack that we needed to do when we added storybook-docs, but our components didn't have README files I think

(I will update the plop file to not have this line someday)

Reply:
https://github.com/contentful/forma-36/pull/1207
Created a PR removing those lines and adding the `Component.mdx` file

Reply:
Legend! 🙌